operator-common/src/main/java/io/strimzi/operator/common/MicrometerMetricsProvider.java

```diff
@@ -74,11 +75,32 @@ public Counter counter(String name, String description, Tags tags) {
     public Timer timer(String name, String description, Tags tags) {
         return Timer.builder(name)
                 .description(description)
-                .sla(Duration.ofMillis(1000), Duration.ofMillis(5000), Duration.ofMillis(10000), Duration.ofMillis(30000), Duration.ofMillis(60000), Duration.ofMillis(120000), Duration.ofMillis(300000))
+                .sla(Duration.ofMillis(1000),
+                    Duration.ofMillis(5000),
+                    Duration.ofMillis(10000),
+                    Duration.ofMillis(30000),
+                    Duration.ofMillis(60000),
+                    Duration.ofMillis(120000),
+                    Duration.ofMillis(300000))
```

Comment:
when I did something similar for the canary component, I made the buckets configurable. Doesn't it make sense having the same here?

Reply:
That's a good point, which was also raised by Tom. Let me work on that.